SR2020/2019-10-21-kit-disclaimer-form.md

```diff
@@ -0,0 +1,22 @@
+---
+to: sr2020-teams
+subject: Kit Disclaimer Forms
+attachments:
+  - https://drive.google.com/a/studentrobotics.org/file/d/1aCHnZ6o8DkLzHef1C2lz_iIkZkOjYP6P/view?usp=sharing
+---
+
+Hi all,
+
+Kickstart is less than a week away, and we're putting the finishing touches to kickstart. If you're attending a kickstart event, your kits are ready and waiting at the venue. For those who can't attend kickstart, we'll contact you shortly to confirm details of shipping your kits.
+
+Before we can issue a kit, you must sign a kit disclaimer form. Historically these have been signed at the event, but we're sending them over early to speed up the issuing process on the day. If you're not attending Kickstart, return your form promptly to ensure you receive your kit as soon afterwards as possible.
+
+Attached to this email is the disclaimer form. Please read it, print it, sign it, and [send it back to us](mailto:teams@studentrobotics.org). Alternatively, feel free to sign it and bring it with you. We will of course have some spares at the event to sign on the day.
+
+Please only attend the kickstart we're expecting you at: https://docs.google.com/spreadsheets/d/1plZqwcO0WIdoWhiUNdGxy-6uLSoH8vIsFdKzwHXtf0c/edit?usp=sharing. If you can no longer attend, please let us know.
+
+Last week, we sent out your login details for our account management system. This will enable you to add competitors to give them access to the IDE and forums. Please check you can login and create competitors before kickstart.
```

Comment:
Perhaps explicitly explain why this is important -- that the competitors will need to use the IDE and forums at Kickstart.